<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HP" failures="1" tests="1" name="FirefoxTest" time="38.604" errors="0" timestamp="2025-07-17T16:25:29 AST">
  <testcase classname="testCases.TC001Enrolling" name="verify_eroll" time="0.727">
    <failure type="java.lang.AssertionError" message="expected [test] but found [test1]">
      <![CDATA[java.lang.AssertionError: expected [test] but found [test1]
at testCases.TC001Enrolling.verify_eroll(TC001Enrolling.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- verify_eroll -->
</testsuite> <!-- FirefoxTest -->
